pio-scripts/validate_usermods.py

```
@@ -0,0 +1,92 @@
+import re
+import sys
```

Ruff (0.11.9)

`sys` imported but unused

Remove unused import: `sys`

(F401)